worker/test/src/RTC/Codecs/TestDependencyDescriptor.cpp

```diff
@@ -0,0 +1,100 @@
+#include "common.hpp"
+#include "RTC/Codecs/DependencyDescriptor.hpp"
+#include <catch2/catch_test_macros.hpp>
+
+using namespace RTC;
+
+SCENARIO("parse Dependency Descriptor", "[codecs][DD]")
+{
+	SECTION("parse Dependency Descriptor")
+	{
+		/**
+		 * Taken from https://issues.webrtc.org/issues/42225660.
```

Comment:
Can we please use /** */ comment syntax with all the lines in the comment prefixed with *?